examples/common/python/connectors/direct/unit_tests/test_worker_registry_jrpc_impl.py

```diff
@@ -20,7 +20,7 @@
 import secrets
 import json
 
-from tcf_connector.worker_registry_jrpc_impl import WorkerRegistryJRPCImpl
+from connectors.direct.worker_registry_jrpc_impl import WorkerRegistryJRPCImpl
 from utility.tcf_types import WorkerType, WorkerStatus
 
```

Comment:
Update the toml file path. It is pointing to old path tcf_connector..

Reply:
Done.